torch/_prims/utils.py

```diff
@@ -217,6 +219,16 @@ def validate_shape(shape: Sequence):
         validate_dim_length(l)
 
 
+def validate_strides(strides: StrideType):
+    """
+    Verifies the object specifies valid strides.
+    """
+
+    assert isinstance(strides, Sequence)
+    for stride in strides:
+        assert stride >= 0
```

Comment:
do we have general rules on assert vs RuntimeError? OpInfo tests expect RuntimeError usually

Reply:
Yeah we should convert these asserts